cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -47,9 +48,39 @@ AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer) : APawn
 void AUrdfBotPawn::SetupPlayerInputComponent(class UInputComponent* input_component)
 {
     Super::SetupPlayerInputComponent(input_component);
+
+    int axis_binding_index = 0;
+    UPlayerInput* player_input = GetWorld()->GetFirstPlayerController()->PlayerInput;
+    std::vector<YAML::Node> keyboard_controls_ = Config::get<std::vector<YAML::Node>>("URDFBOT.URDFBOT_PAWN.ROBOT_COMPONENT.KEYBOARD_CONTROL");
+    for (auto& control : keyboard_controls_) {
+        InputAxisBinding input_axis_binding;
```

Comment:
KeyboardAction instead of InputAxisBinding

Reply:
updated. the axis still uses the type of `FName` because so far all places that uses it need `FName` and I dont think we will need it anywhere else.

Reply:
slightly changed YAML format for cleaner parsing code: 
```
    KEYBOARD_ACTIONS:
      SPACEBAR:
        APPLY_ACTION: { "r_wheel_joint": 0,"l_wheel_joint": 0 }
      UP:
        APPLY_ACTION: { "r_wheel_joint": 1,"l_wheel_joint": 1 }
      Q:
        ADD_ACTION: { "joint_1": 1 }
      E:
        ADD_ACTION: { "joint_1": -1 }
```

Reply:
I think the name of the C++ struct should be KeyboardAction.

Reply:
fixed